modules/openapi-generator/src/test/java/org/openapitools/codegen/java/AbstractJavaCodegenTest.java

```diff
@@ -42,27 +42,27 @@
 
     @Test
     public void toEnumVarNameShouldNotShortenUnderScore() throws Exception {
-        Assert.assertEquals("UNDERSCORE", fakeJavaCodegen.toEnumVarName("_", "String"));
-        Assert.assertEquals("__", fakeJavaCodegen.toEnumVarName("__", "String"));
-        Assert.assertEquals("__", fakeJavaCodegen.toEnumVarName("_,.", "String"));
+        Assert.assertEquals(fakeJavaCodegen.toEnumVarName("_", "String"), "UNDERSCORE");
+        Assert.assertEquals(fakeJavaCodegen.toEnumVarName("__", "String"), "__");
+        Assert.assertEquals(fakeJavaCodegen.toEnumVarName("_,.", "String"), "__");
     }
 
     @Test
     public void toVarNameShouldAvoidOverloadingGetClassMethod() throws Exception {
-        Assert.assertEquals("propertyClass", fakeJavaCodegen.toVarName("class"));
-        Assert.assertEquals("propertyClass", fakeJavaCodegen.toVarName("_class"));
-        Assert.assertEquals("propertyClass", fakeJavaCodegen.toVarName("__class"));
+        Assert.assertEquals(fakeJavaCodegen.toVarName("class"), "propertyClass");
+        Assert.assertEquals(fakeJavaCodegen.toVarName("_class"), "propertyClass");
+        Assert.assertEquals(fakeJavaCodegen.toVarName("__class"), "propertyClass");
```

Comment:
All above changes are minor cleanup. The assertions weren't in `(actual, expected)` format, which could lead to confusion on build failures.